editor/translations/el.po

```diff
@@ -6,7 +6,7 @@
 # Georgios Katsanakis <geo.elgeo@gmail.com>, 2019.
 # Overloaded <manoschool@yahoo.gr>, 2019.
 # Eternal Death <eternaldeath0001@gmail.com>, 2019.
-# Overloaded @ Orama Interactive http://orama-interactive.com/ <manoschool@yahoo.gr>, 2020.
+# Overloaded @ Orama Interactive https://orama-interactive.com/ <manoschool@yahoo.gr>, 2020.
```

Comment:
This is also wrong, that's the identity that @OverloadedOrama uses on Weblate, we shoudn't be modifying it.

Reply:
For the record Orama did reply to me via email that it's fine to use HTTPS for them if we want to.

Reply:
Oops, I missed the tag. Feel free to remove that line completely, I have changed my Weblate nickname and removed the website URL completely, and now I appear in el.po twice 😅